app/inventory/batch/template.hbs

```diff
@@ -10,7 +10,7 @@
       {{select-or-typeahead property="aisleLocation" label=(t 'inventory.labels.aisle_location') list=aisleLocationList selection=model.aisleLocation className="col-sm-5"}}
       <div class="form-group col-sm-2">
         <label class="control-label">{{t 'inventory.labels.gift'}}</label>
-        {{em-checkbox label=(t 'inventory.labels.gift') property="giftInKind"}}
```

Comment:
I think we should probably keep the label text on the element and :fire: the label element on the line above because not having the text immediately beside the label makes it less clear what text the checkbox is associated with.